### DISCUSSION TITLE
[Idea] 分析代码仓库中代码

### DISCUSSION THREAD

**Discussion by Royalvice** [lobehub/lobehub#771] [Ideas]
### 🥰 需求描述 | Feature Description

通过上传一个代码仓库（小型的）或者github链接，分析代码结构，方便开发者快速看懂代码逻辑。

### 🧐 解决方案 | Proposed Solution

插件支持

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
我也很想要

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
https://sharegpt.com/c/6XKZuNZ

**Reply by ccdanpian** [reply]
chatgpt上有个askthecode的插件，就可以提供给他github链接分析，是否可以接入这个插件？

我是一个软件初学者，昨天用这个插件学习了一下网络爬虫插件怎么实现的。

**Reply by arvinxx** [reply]
> chatgpt上有个askthecode的插件，就可以提供给他github链接分析，是否可以接入这个插件？
> 
> 我是一个软件初学者，昨天用这个插件学习了一下网络爬虫插件怎么实现的。

可以share一下会话记录吗？我想了解下这个插件的回复是什么样的

**Reply by ccdanpian** [reply]
@arvinxx https://chat.openai.com/share/bfcc0371-e4dc-4780-9183-a3a324c4732b
